spikeextractors/recordingextractor.py

```diff
@@ -496,11 +496,20 @@ def copy_channel_properties(self, recording, channel_ids=None):
         else:
             if isinstance(channel_ids, (int, np.integer)):
                 channel_ids = [channel_ids]
+
```

Comment:
This is because otherwise key_properties are duplicated as normal properties as well.

Reply:
see https://github.com/SpikeInterface/spiketoolkit/issues/411